lib/quizBrain.dart

```diff
@@ -31,6 +42,7 @@ class QuizBrain {
         true),
   ];
 
+  //SUGGESTION: maybe if you could change the name checker to a method name like goToNextQuestion would make this amazing of a code even better.
   void checker() {

```

Comment:
Same suggestion as before: maybe if you use a more understandable name like moveToNextQuestion